pkg/repo/index_test.go

```diff
@@ -29,7 +29,12 @@ func TestDownloadIndexFile(t *testing.T) {
 		fmt.Fprintln(w, string(fileBytes))
 	}))
 
-	dirName, err := ioutil.TempDir("testdata", "tmp")
+	dirName, err := ioutil.TempDir("", "tmp")
```

Comment:
@michelleN Is this okay with you? This will create it in `/tmp` instead of here in the `testdata/` directory.